### PR TITLE
[WEB-161]Update Atlas/Layer lib versions to: Atlas(0.4.0), Layer(0.23.6)

### PR DESCRIPTION
**Pull request by CodeFactoryPDabrowski**
https://netguru.atlassian.net/browse/WEB-161
Prepare newest version of Atlas 0.4.0 to work with Participant object(instead of full integration with Identity object).

### CODE REVIEW

File: layer-atlas/src/main/java/com/layer/atlas/AtlasAddressBar.java [hunk 1]
```diff
@@ -516,77 +513,91 @@ protected void onPostExecute(List<Identity> identities) {
...
-                return new HashSet<>();
+        private void searchParticipantsReady(Set<Participant> selectedParticipants, Map<String, Participant> participants) {
+            Identity authenticatedUser = mLayerClient.getAuthenticatedUser();
+            final String userId = authenticatedUser != null ? authenticatedUser.getUserId() : null;
+            mParticipantMap.clear();
+            mParticipantMap.putAll(participants);
+            mParticipants.clear();
+            for (Map.Entry<String, Participant> entry : mParticipantMap.entrySet()) {
+                // Don't show participants we've already selected
+                if (selectedParticipants.contains(entry.getKey())) {
```
Comment (MarcinGrabowski91) [inline layer-atlas/src/main/java/com/layer/atlas/AtlasAddressBar.java:577]: Why not || operator?

File: layer-atlas/src/main/java/com/layer/atlas/typingindicators/AvatarTypingIndicatorFactory.java [hunk 2]
```diff
@@ -83,48 +97,69 @@ public LinearLayout onCreateView(Context context) {
     }
 
     @Override
-    public void onBindView(LinearLayout l, Map<Identity, LayerTypingIndicatorListener.TypingIndicator> typingUserIds) {
-        @SuppressWarnings("unchecked")
+    public void onBindView(final LinearLayout l, final Map<Identity, LayerTypingIndicatorListener.TypingIndicator> typingUserIds) {
+        @SuppressWarnings("unchecked") final
```
Comment (MarcinGrabowski91) [inline layer-atlas/src/main/java/com/layer/atlas/typingindicators/AvatarTypingIndicatorFactory.java]: final to lower line

**Review by MarcinGrabowski91 — APPROVED** [review]
Few small comments

File: layer-atlas/src/main/java/com/layer/atlas/AtlasAddressBar.java (hunk 1 above)
Comment (CodeFactoryPDabrowski) [reply layer-atlas/src/main/java/com/layer/atlas/AtlasAddressBar.java:577]: Layer implementation, didn't change this :(

File: layer-atlas/src/main/java/com/layer/atlas/typingindicators/AvatarTypingIndicatorFactory.java (hunk 2 above)
Comment (CodeFactoryPDabrowski) [reply layer-atlas/src/main/java/com/layer/atlas/typingindicators/AvatarTypingIndicatorFactory.java]: done

**Review by MarcinGrabowski91 — APPROVED** [review]

**Pull request merged**